californium-core/src/main/java/org/eclipse/californium/core/network/stack/ReliabilityLayer.java

```diff
@@ -50,15 +49,16 @@
 
 	/** The random numbers generator for the back-off timer */
 	private Random rand = new Random();
-	
+
 	private int ack_timeout;
 	private float ack_random_factor;
 	private float ack_timeout_scale;
 	private int max_retransmit;
 
 	/**
-	 * Constructs a new reliability layer.
-	 * Changes to the configuration are observed and automatically applied.
+	 * Constructs a new reliability layer. Changes to the configuration are
+	 * observed and automatically applied.
```

Comment:
Actually, I think we should get rid of this, i.e. we should not observe the Configuration object for changes. IMHO the configuration should be static through the run-time of an Endpoint. If the config is supposed to be changed then the endpoint needs to be stopped and re-created with a new config object.

WDYT?